pkg/connector/connector.go

```diff
@@ -27,6 +27,11 @@ var (
 		DisplayName: "Team",
 		Traits:      []v2.ResourceType_Trait{v2.ResourceType_TRAIT_GROUP},
 	}
+	resourceTypeEscalation = &v2.ResourceType{
```

Comment:
I would think we'd need to add a resource builder for this type, otherwise we'll be referencing unknown resources in our grants.

Reply:
I see, I am going to just remove this for now since we continue on escalation types.